static/templyte/src/app/tutorial/tutorial.component.html

```diff
@@ -0,0 +1,82 @@
+<mat-toolbar class="mat-elevation-z3">
+    <nav class="navbar">
+        <ul class="nav navbar-right">
+            <li>
+                <button mat-button (click)="goHome()">Home</button>
+            </li>
+            <li>
+                <button mat-button (click)="createTemplate()">Create Template</button>
+            </li>
+            <li>
+                 <button mat-button disabled>Tutorial</button>
+            </li>
+            <li>
+                <button mat-button (click)="myTemplates()">My Templates</button>
+            </li>
+            <li>
+                <button mat-button (click)="logout()">Logout</button>
+            </li>
+        </ul>
+    </nav>
+</mat-toolbar>
+
+<html>
+<head>
+
+<title>Page Title</title>
+</head>
+<body>
+<h1>Create a Template</h1>
+
```

Comment:
I think it would be a good idea to add some steps in that describe actually writing the template and making the zip file. Just tell them to put curly braces around variable names and to zip up all the relevant files.